src/components/forms/contactUs.tsx

```diff
@@ -70,39 +82,71 @@ const ContactUsForm: React.FC = () => {
           }, 5000);
         });
     },
-    [reload, reset]
+    [intl, reload, reset]
+  );
+
+  const inputLabels = useMemo(
+    () => ({
+      name: intl.formatMessage({
+        id: 'section.contact-us-form.input-field.label.name',
+        defaultMessage: 'Name',
+      }),
+      service: intl.formatMessage({
+        id: 'section.contact-us-form.input-field.label.service',
+        defaultMessage: 'Service',
+      }),
+    }),
+    [intl]
   );
 
   return (
     <div className="pt-5 mx-auto md:w-2/3" id="contact-us">
       <form className="flex flex-col gap-5" onSubmit={handleSubmit(onSubmit)}>
         <div>
           <TextInput
-            placeholder="Name"
-            {...register('name', { required: 'Please enter a name' })}
+            placeholder={inputLabels.name}
+            {...register('name', {
+              required: intl.formatMessage({
+                id: 'section.contact-us-form.input-field.empty-error.name',
+                defaultMessage: 'Please enter a name',
+              }),
+            })}
+            name="name"
             error={errors.name?.message}
           />
         </div>
         <div>
           <TextInput
             placeholder="yourname@example.com"
             {...register('email', {
-              required: 'The email is required',
+              required: intl.formatMessage({
+                id: 'section.contact-us-form.input-field.empty-error.email',
+                defaultMessage: 'The email is required',
+              }),
               pattern: {
                 value:
                   /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/,
-                message: 'Please enter a valid email',
+                message: intl.formatMessage({
+                  id: 'section.contact-us-form.input-field.pattern-error.email',
+                  defaultMessage: 'Please enter a valid email',
+                }),
               },
             })}
+            name="email"
             error={errors.email?.message}
           />
         </div>
         <div>
-          <Label name="service" />
+          <Label name="service" label={inputLabels.service} />
```

Comment:
```suggestion
          <Label label={inputLabels.service} />
```
I'm not sure tho why this has a label prop in the first place. I would suggest just using the child element

Reply:
using children now